product/pom.xml

```diff
@@ -312,10 +346,10 @@
                                     <id>org.wso2.carbon.datasource.core.feature</id>
                                     <version>${carbon.datasources.version}</version>
                                 </feature>
-                                <feature>
-                                    <id>org.wso2.carbon.security.userstore.jdbc.feature</id>
+                                <!--feature>
```

Comment:
Rather than commenting-out, if no longer used then let's remove this feature entry.